cmd/erasure.go

```diff
@@ -197,6 +197,7 @@ func getDisksInfo(disks []StorageAPI, endpoints []string) (disksInfo []madmin.Di
 				RootDisk:       info.RootDisk,
 				Healing:        info.Healing,
 				State:          diskErrToDriveState(err),
+				FreeInodes:     info.Free - info.UsedInodes,
```

Comment:
Huh? Something is wrong. Either here or `info.Free` isn't the number of free inodes.

Reply:
Fixed